### ISSUE TITLE
Keychron Q2 Pro not detected

### ISSUE THREAD

**Issue by notkoyo** [the-via/releases#278]
I've tried connecting and reconnecting my board, importing the .json file directly from the VIA GitHub, tried both the non-knob version and knob version (i have the knob version if that makes a difference) and it still won't work, ive tried using the web version and v2.2.0 and v3.0.0 of the app. Nothing works.

**Comment by ix-56h** [comment]
Same here.

**Comment by adophoxia** [comment]
Was loading the JSON done while the board was connected via wired with the power toggle set to `Cable` mode?

**Comment by notkoyo** [comment]
Yes i tried while it was connected and connecting it after loading the file too.

**Comment by giuli007** [comment]
same here
